docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,209 @@
+---
+approvers:
+- davidopp
+- wojtek-t
+title: Pod Priority and Preemption (Alpha)
+---
+
+[Pods](/docs/user-guide/pods) in Kubernetes 1.8 and later can have priority. Priority
+indicates importance of a pod relative to other pods. When a pod cannot be scheduled, scheduler tries
+to preempt (evict) lower priority pods in order to make scheduling of the pending pod possible.
+In a future Kubernetes release, priority will also affect out-of-resource eviction ordering on the node.
+
+Note that preemption does not respect PodDisruptionBudget; see 
+[the limitations section](#poddisruptionbudget-is-not-supported) for more details.
+
+* TOC
+{:toc}
+
+## How to use it
+In order to use priority and preemption in Kubernetes 1.8, you should follow these
+steps:
+
+1. Enable the feature.
+1. Add one or more PriorityClasses.
+1. Create pods with `PriorityClassName` set to one of the added PriorityClasses.
+(Of course you do not need to create the pods directly; normally you would add 
+`PriorityClassName` to the pod template of the collection object managing your 
+pods, for example a Deployment.)
+
+The following sections provide more information about these steps.
+
+## Enable Priority and Preemption
+Pod priority and preemption is disabled by default in Kubernetes 1.8 as it is an
+__alpha__ feature. It can be enabled by a command-line flag for API server and scheduler:
+
+```
+--feature-gates=PodPriority=true
+```
+and also the following command-line flag for API server:
+```
+--runtime-config=scheduling.k8s.io/v1alpha1=true
+```
+
+Once enabled you can add [PriorityClasses](#priorityclass) and create pods with [`PriorityClassName`](#pod-priority) set.
+If you tried it and decided to disable it, you must remove this command-line flag or
+set it to false and restart API server and Scheduler. Once disabled, the existing
+pods will keep their priority fields, but preemption will be disabled and priority
+fields will be ignored, and you will not be able to set PriorityClassName in new pods.
+
+**Note:** Alpha features should not be used in production systems! Alpha 
+features are more likely to have bugs and future changes to them are not guaranteed to
+be backward compatible.
```

Comment:
> not guaranteed to be backward compatible.

Do we have doc on the scope of alpha, beta and GA?

Reply:
em... maybe you are referring to this? https://kubernetes.io/docs/reference/deprecation-policy/

Reply:
@k82cn This doc tries to state the features/improvements planned for Beta, but we don't have any other doc at this point.